.coveragerc

```diff
@@ -0,0 +1,2 @@
+[run]
+branch = True
```

Comment:
Normally a `--branches` option could be sent to pytest upon each CLI invocation (or `--cover-branches` when using nose).  However, it looks like `pytest-cov` does not support passing command-line options such as branch coverage (see this [issue #1][pytest-cov 1] on the old pytest-cov issue tracker).

So without forking pytest-cov, it looks like the only way to pass options to coverage is via a configuration file.  It may be possible to specify the configuration file location via the command line using the [`--cov-config` option. ][cov-config] I will look into that.

[pytest-cov 1]: https://bitbucket.org/memedough/pytest-cov/issues/1/allow-branch-coverage
[cov-config]: https://github.com/pytest-dev/pytest-cov#coverage-config-file